daemon/src/routes_maker.rs

```diff
@@ -39,11 +38,11 @@ pub async fn maker_feed(
         let order = rx_order.borrow().clone();
         yield order.to_sse_event();
 
-        let cfds = rx_cfds.borrow().clone();
-        yield cfds.to_sse_event();
         let quote = rx_quote.borrow().clone();
         yield quote.to_sse_event();
 
+        let cfds_with_price = CfdsWithCurrentPrice{cfds: rx_cfds.borrow().clone(), current_price: quote.for_maker()};
```

Comment:
This formatting is a bit off but not being picked up by dprint due to the macro, can you format it manually please?

Reply:
thanks for spotting! I sometimes still forget about these macro problems. fixed in all occurrences & pushed a fixup commit.